pymllm/tests/test_scheduler_cpu_busy_loop.py

```
@@ -0,0 +1,302 @@
+            proc.event_loop()
+        except StopIteration:
+            pass
+
+        # call_log should be:
+        # iter 0: recv(brief_poll=False) → extend batch → _in_decode=False
+        # iter 1: recv(brief_poll=False) → decode batch → _in_decode=True
+        # iter 2: recv(brief_poll=True)  → None → _in_decode=False
+        # iter 3: recv(brief_poll=False) → StopIteration
+        assert call_log[0] == ("recv", False), f"iter 0: {call_log[0]}"
+        assert call_log[1] == ("recv", False), f"iter 1: {call_log[1]}"
+        assert call_log[2] == ("recv", True), f"iter 2: should be True after decode"
```

Ruff (0.15.6)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)